cli/tests/unit/headers_test.ts

```diff
@@ -372,22 +372,12 @@ unitTest(function headersAppendDuplicateSetCookieKey(): void {
   headers.append("Set-cookie", "baz=bar");
   const actual = [...headers];
   assertEquals(actual, [
+    ["set-cookie", "foo=bar"],
     ["set-cookie", "foo=baz"],
     ["set-cookie", "baz=bar"],
   ]);
 });
 
-unitTest(function headersSetDuplicateCookieKey(): void {
-  const headers = new Headers([["Set-Cookie", "foo=bar"]]);
-  headers.set("set-Cookie", "foo=baz");
-  headers.set("set-cookie", "bar=qat");
-  const actual = [...headers];
-  assertEquals(actual, [
-    ["set-cookie", "foo=baz"],
-    ["set-cookie", "bar=qat"],
-  ]);
-});
```

Comment:
This test is plain wrong. `headers.set` is not the same as append. It removes existing headers of that name before insertion. See https://fetch.spec.whatwg.org/#concept-header-list-set